docs/synthetics/ssl-certs.md

```diff
@@ -0,0 +1,66 @@
+title: SSL Certificate Monitoring
+description: Monitor & get notified on SSL certificate expiry and change. 
+
+Issues with SSL certificates not only cause downtime of APIs and websites but also hurts their reputation. Sematext Synthetics performs the following SSL certificate checks on all the certificates in the chain - the leaf, intermediate, and root certificates.
+
+* Validate the SSL certificates of the API/websites in every run
+* Check for the expiration of the certificates every day and alert you 30, 14, 7 and 3 days before the expiry
+* Check for certificate change every 10 mins and alert you on detecting any changes, with a detailed change report
+
+Sematext Synthetics alerts you on the failure of these checks via the configured monitor alert channels like Email, Slack, Webhooks, etc. Apart from these checks, Synthetics also provides an SSL certificate report, with details of all the certificates in the chain.
```

Comment:
"monitor alert channels" ==> "notification hooks". We can link that to https://sematext.com/docs/integration/#notification-hooks

Reply:
Changed to `alert notification hooks` and linked to https://sematext.com/docs/alerts/alert-notifications/